cmd/web/tmpl/showpaste.tmpl

```diff
@@ -1,11 +1,18 @@
 {{template "header" .}}
 
+<link href="/static/css/prism.css" rel="stylesheet" />
```

Comment:
should prob be in `<head>` no?

Reply:
Yes, however that is logistically inconvenient for how this program works. I could add it to the header, but I would have to rejigger how the templates work